src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-mouseposition-control.ts

```diff
@@ -26,7 +26,7 @@ export class OpenLayersMousePositionControl extends MousePosition {
 	}
 
 	private updateHtmlIfNeeded(html) {
-		if (!this.renderedHTML_ || html != this.renderedHTML_) {
+		if (!this.renderedHTML_ || html !== this.renderedHTML_) {
```

Comment:
Why did you make this change? First, it's beyond the scope of this PR. Second, since both are strings there is no difference between the operators.

Reply:
`ts-lint`